data/hai/hai reveal 4 middle.txt

```diff
@@ -72,9 +72,9 @@ mission "Hai Reveal [A02] Symbol Identification"
 				`	(Walk into the cave.)`
 				`	(Come back later.)`
 					defer
-			`	You walk twenty meters into the cave and enter an open doorway before passing through a presently unmanned security post into a small and roughly designed underground facility. There's a variety of medical machinery in rooms with large glass windows immediately to the left, and several computers arrayed around a makeshift living area with what looks to be some sleeping quarters off to the back and right. Some of the people you've rescued are present, though there are many more humans and Hai that you haven't met. A few of them have pulse rifles, but your attention is quickly captured by a woman walking towards you.`
+			`	You walk twenty meters into the cave and enter an open doorway before passing through an unmanned security post into a small and roughly designed underground facility. There's a variety of medical machinery in rooms with large windows immediately to the left, and computers arrayed around a makeshift living area with sleeping quarters off to the right. Some people you've rescued are present, with many humans and Hai you haven't met. Some have pulse rifles, but your attention is quickly captured by a woman walking towards you.`
```

Comment:
```suggestion
			`	You walk twenty meters into the cave before coming up to a concrete wall that fills up the entirety of the cavity. An open door passes through to an unmanned security post, until it finally leads into a small and roughly designed underground facility. Through some large windows immediately to the left, you can see a room containing a variety of medical machinery, while off to the right there are computers arrayed around a makeshift living area and some sleeping quarters. You notice that some of the people you've rescued are present, but there are many more humans and Hai you haven't met, some armed with pulse rifles. Despite all this, your attention is quickly captured by a Hai woman walking towards you.`
```

I think that some of these changes made less sense than before, although it's shorter. I'm going to add a little bit of detail that fills some gaps, because I had a hard time imagining the facility before, but it does change it a bit. MasterOfGrey will have to take a look at this.

Reply:
Okay, then I'll delay applying this change until we see his comment.

Reply:
I don't think details of the facility are important in their own right. That text needs to make the points that the facility is compact, well-hidden, and highly capable, because that is what is important to the story. It should give atmosphere too. Anything beyond just contributes text length, not substance.

Reply:
Fair enough, but I don't see how the previous edit makes the point especially well. And if I can't imagine what is going on, I feel that understanding the story is harder.

Reply:
I'm going to accept your change because I want this PR just to be a dialog rearrangement, with no changes to the story. Changing from "cadmium poisoning" feels like a story change.